### PR TITLE
[test]Fix index after load for chaos test

### PR DESCRIPTION
**Pull request by zhuwenxing**
Signed-off-by: zhuwenxing <wenxing.zhu@zilliz.com>

/kind improvement

### CODE REVIEW

File: tests/python_client/chaos/testcases/test_all_collections_after_chaos.py [hunk 1]
```diff
@@ -51,15 +51,26 @@ def test_milvus_default(self, collection_name):
...
-        t0 = time.time()
-        index, _ = collection_w.create_index(field_name=ct.default_float_vec_field_name,
-                                             index_params=_index_params,
-                                             name=cf.gen_unique_str())
-        tt = time.time() - t0
-        log.info(f"assert index: {tt}")
+        # create index if not have
+        index_infos = [index.to_dict() for index in collection_w.indexes]
+        index_params = {"index_type": "HNSW", "metric_type": "L2", "params": {"M": 48, "efConstruction": 500}}
+        if len(index_infos) == 0:
```
[the remark this answers is not in the document]
Comment (zhuwenxing) [reply tests/python_client/chaos/testcases/test_all_collections_after_chaos.py:57]: I will fix this corner case in the next PR. For now, I will not just create an index only for the scalar field.
If `len(index_infos) > 0`, it means at least the vector index is created for now.
